Sources/PubNub/Helpers/Constants.swift

```diff
@@ -62,13 +62,7 @@ public struct Constant {
   }()
 
   static let pubnubSwiftSDKVersion: String = {
-    guard let pubnubInfo = Bundle(for: HTTPSession.self).infoDictionary,
-      let build = pubnubInfo["CFBundleShortVersionString"]
-    else {
-      return "?.?.?"
-    }
-
-    return "\(build)"
+    "4.0.2"
```

Comment:
@CraigLn had to do this, because old approach doesn't work in case of usage through SPM.
Version bump will be handled by deployment scripts.

Reply:
Awesome!  I completely forgot about this in the last couple changes I made, so thank you for getting this one in.